CHANGES.rst

```diff
@@ -4,7 +4,7 @@
 
 New Features
 ^^^^^^^^^^^^
-
+- Add Representation for ImageFileCollection []
```

Comment:
You'll need to fill in the PR number. Also no need to capitalize `representation` (I guess, not sure about that).